framework/retryresource/resource.go

```diff
@@ -135,7 +140,7 @@ func (r *Resource) GetCreateState(obj, currentState, desiredState interface{}) (
 	}
 
 	n := func(err error, dur time.Duration) {
-		r.logger.Log("warning", fmt.Sprintf("retrying 'GetCreateState' due to error (%s)", err.Error()))
+		r.logger.Log("warning", fmt.Sprintf("retrying 'GetCreateState' of resource '%s' due to error (%s)", r.Origin().Name(), err.Error()))
```

Comment:
As I see it the origin/underlying resource is the one that matters most to the user and that might be more relevant to see information about in the logs. Consider a wrapped chain like this: `retry` -> `metrics` -> `configmap`. `configmap` is for instance a resource that implements magic about ingress-operator configmaps. This is the main logic we care about. This is the origin/underlying resource. `metrics` wraps `configmap`. `retry` wraps `metrics`. In case we only use `r.Name()` here the log message of the retry resource would reference the metrics resource, which is always the same and thus quite useless. Using `r.Origin().Name()` the logs of the retry resource refer to `configmap` which is the specific resource that is actually being retried. 

Note that this example might be constructed now but it shows how it works and why we should probably favour `r.Origin().Name()` over `r.Name()`.